rescript/merge.py

```diff
@@ -22,7 +22,8 @@
 def merge_taxa(data: pd.DataFrame,
                mode: str = 'len',
                rank_handle_regex: str = '^[dkpcofgs]__',
-               new_rank_handle: str = None,
```

Comment:
did you intend to change the default? I am fine with this, but just want to make sure — it should be documented in the release notes in the next tagged release of rescript